core/src/main/java/jenkins/model/RunIdMigrator.java

```diff
@@ -277,30 +278,8 @@ private void doMigrate(File dir) {
      * TODO candidate for moving to {@link Util}
      */
     static void move(File src, File dest) throws IOException {
-        Class<?> pathC;
         try {
-            pathC = Class.forName("java.nio.file.Path");
-        } catch (ClassNotFoundException x) {
-            // Java 6, do our best
-            if (dest.exists()) {
-                throw new IOException(dest + " already exists");
-            }
-            if (src.renameTo(dest)) {
-                return;
-            }
-            throw new IOException("could not move " + src + " to " + dest);
-        }
-        try {
-            Method toPath = File.class.getMethod("toPath");
-            Class<?> copyOptionAC = Class.forName("[Ljava.nio.file.CopyOption;");
-            Class.forName("java.nio.file.Files").getMethod("move", pathC, pathC, copyOptionAC).invoke(null, toPath.invoke(src), toPath.invoke(dest), Array.newInstance(copyOptionAC.getComponentType(), 0));
-        } catch (InvocationTargetException x) {
-            Throwable cause = x.getCause();
-            if (cause instanceof IOException) {
-                throw (IOException) cause;
-            } else {
-                throw new IOException(cause);
-            }
+            Files.move(src.toPath(), dest.toPath());
```

Comment:
catch IOException and rethrow to avoid wrapping an IOException with an IOException